package.json

```diff
@@ -101,7 +101,7 @@
         "text-transform-loader": "^2.0.0",
         "ts-node": "~3.2.0",
         "tslint": "~5.7.0",
-        "typescript": "~2.3.3",
+        "typescript": "^2.5.3",
```

Comment:
The version of Typescript supported by Angular 5 is 2.4.2, see [changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md) (commit: [ca5aeba](https://github.com/angular/angular/commit/ca5aeba))

Reply:
The compiler requires 2.4, and It is actually set to `"typescript": ">=2.4.2 <2.5",` for now, because there is still testing to be done on 2.5 or 2.6 (not that there is a known reason it fails to my knowledge).

Reply:
ok

Reply:
see: https://github.com/angular/angular/issues/19587#issuecomment-337713553